README.md

```diff
@@ -49,4 +49,4 @@ Name | Tags
 - [Programiz](https://www.programiz.com/blog/)
 - [Yarsa Labs](https://blog.yarsalabs.com/)
 - [Young Innovations](https://blog.yipl.com.np/)
-
+- [Rara Labs](https://medium.com/@raralabs)
```

Comment:
hey @ankurgajurel the names are alphabetically sorted, can you put it after Programiz?